wire/gen_wire_csv

```diff
@@ -51,9 +51,6 @@ funding_locked,8,channel-id,8
 funding_locked,16,announcement-node-signature,64
 funding_locked,80,announcement-bitcoin-signature,64
 funding_locked,144,next-per-commitment-point,33
-update_fee,37
-update_fee,0,channel-id,8
-update_fee,8,feerate-per-kw,4
 shutdown,38
```

Comment:
Yeah, it was picked up when pulling changes from the lightningnetwork/lightning-rfc repo. I'll split it into its own commit.